fxmanifest.lua

```diff
@@ -6,7 +6,11 @@ version '1.0.0'
 
 ui_page 'html/index.html'
 
-shared_script 'config.lua'
+shared_scripts {
+    '@qb-core/shared/locale.lua',
+    'locales/*.lua',
```

Comment:
```suggestion
    'locales/en.lua',
    'locales/*.lua',
```
`locales/en.lua` is also needed here.